src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -55,18 +61,20 @@ def next(self):
     def __next__(self):
```

Comment:
No typing on `__next__`? In general, we should add typing on all public functions, classes, and methods.